schemas/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json

```diff
@@ -189,6 +168,9 @@
         }
       },
       "type": "object"
+    },
+    "WebScience.SurveyId": {
+      "type": "string"
```

Comment:
It's good practice to document the use of this field using the `description` property, which will get inserted into the BigQuery schema.